vcd/datasource_vcd_nsxt_edgegateway_l2_vpn_tunnel.go

```diff
@@ -0,0 +1,92 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdNsxtEdgegatewayL2VpnTunnel() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdNsxtEdgegatewayL2VpnTunnelRead,
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"edge_gateway_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Edge gateway ID for the tunnel",
```

Comment:
Polishing nit. I see you are capitalizing it in other places, so let's do that consistently:

```suggestion
				Description: "Edge Gateway ID for the tunnel",
```

Reply:
Fixed, thanks